excel/src/test/java/tech/tablesaw/io/xlsx/XlsxReaderTest.java

```diff
@@ -135,6 +135,7 @@ public void testSheetIndex() throws IOException {
             .read(XlsxReadOptions.builder("../data/multiplesheets.xlsx").sheetIndex(1).build());
     assertNotNull(table, "No table read from multiplesheets.xlsx");
     assertColumnValues(table.stringColumn("stringcol"), "John", "Doe");
+    assertEquals("multiplesheets.xlsx#Sheet2",table.name(),"table name is different");
 
```

Comment:
This doesn't match the formatting we use elsewhere. I.e. there should be a space after the commas. If you compile on the command line once with `mvn compile` it will install a git pre-commit hook that should automatically do the formatting for you.

I might remove the last parameter `"table name is different"` since I don't think it adds much to the default message

Reply:
sounds good, sorry about that.

Reply:
I did `mvn compile` but not sure if pre-commit was configured, so I manually made the change. Also removing TODO from the PR description